src/components/AffectedOfferingForm.vue

```diff
@@ -115,7 +116,7 @@ const hiddenResolutionOptions = computed(() => {
         :key="trackerIndex"
         class="osim-tracker-card pb-2 pt-0 pe-2 ps-2 bg-dark"
       >
-        <details>
+        <details :id="tracker.uuid">
```

Comment:
Maybe it is out of the scope of this task, but since we have an `uuid` for each tracker, I would use it for the `v-for` key instead of the `trackerIndex`, using the index is not recommended (at least in react)

Reply:
Good idea, will add that.